.github/workflows/changelog-webapp.yaml

```diff
@@ -0,0 +1,23 @@
+name: Generate changelog
+
+jobs:
+  changelog:
+    # Only if the PR develop -> staging is in review
+    if: github.event.pull_request.head.ref == 'develop'
```

Comment:
🔔 Rename `development` to `develop` for `showcase`
This file should be similar to `release.yml` with merge and PR creation